### PR TITLE
Add-GraphPermissions-Managed-Identity

### PR DESCRIPTION
**Pull request by WTorremans**
## New Script
This script adds specified Graph Permissions to a Managed Identity with a specified name

 - [x] Your script is added to a new folder inside the `powershell` subdirectory
 - [x] The folder contains 1 PowerShell file
 - [x] The PowerShell file follows the correct naming convention
 - [x] The folder contains 1 Readme file
 - [x] The readme file is based on the correct template

### CODE REVIEW

File: PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Add-GraphPermissions-Managed-Identity
```
Comment (agtenr) [inline PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.md]: Can you rename the folder/files to Add-GraphPermissionsToManagedIdentity

File: PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.ps1 [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+$oAppRole = $oGraphSpn.AppRoles | Where-Object {($_.Value -in $permissions)}
+
+Connect-MgGraph -TenantId $tenantId -Scopes Application.Read.All, AppRoleAssignment.ReadWrite.All, RoleManagement.ReadWrite.Directory
+
+foreach($AppRole in $oAppRole)
+{
+  $oAppRoleAssignment = @{
+    "PrincipalId" = $oMSI.ObjectId
+    #"ResourceId" = $GraphAppId
```
Comment (agtenr) [inline PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.ps1]: Can this be removed?

**Review by agtenr — CHANGES_REQUESTED** [review]

File: PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.ps1 (hunk 2 above)
Comment (WTorremans) [reply PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.ps1]: Resolved

File: PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.md (hunk 1 above)
Comment (WTorremans) [reply PowerShell/Add-GraphPermissions-Managed-Identity/Add-GraphPermissions-Managed-Identity.md]: Resolved

**Review by agtenr — APPROVED** [review]

**Pull request merged**